concent_api/core/message_handlers.py

```diff
@@ -464,6 +467,9 @@ def handle_send_force_subtask_results(client_message: message.concents.ForceSubt
 
     current_time = get_current_utc_timestamp()
 
+    if Subtask.objects.filter(subtask_id=task_to_compute.compute_task_def['subtask_id']).exists():  # pylint: disable=no-member
```

Comment:
Maybe it would be better to change this and similar places to `try get()/except DoesNotExists` flow, then it would be only single database query.

Reply:
right, done